Remotion/Validation/UnitTests/Validators/GreaterThanValidatorTest.cs

```diff
@@ -61,15 +63,27 @@ public void Validate_WithPropertyValueLessThanComparisonValue_ReturnsSingleValid
       var validationFailures = validator.Validate(propertyValidatorContext).ToArray();
 
       Assert.That(validationFailures.Length, Is.EqualTo(1));
-      //TODO RM-5906: Assert ValidatedObject, ValidatedProperty, ValidatedValue
+      Assert.That(validationFailures[0].ValidatedObject, Is.EqualTo(propertyValidatorContext.Instance));
+      Assert.That(validationFailures[0].ValidatedProperty, Is.EqualTo(propertyValidatorContext.Property));
+      Assert.That(validationFailures[0].ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
       Assert.That(validationFailures[0].ErrorMessage, Is.EqualTo("The value must be greater than '3'."));
       Assert.That(validationFailures[0].LocalizedValidationMessage, Is.EqualTo("Custom validation message: '3'."));
     }
 
     [Test]
-    [Ignore("RM-5906")]
     public void Validate_WithIComparable_CallsCompareTo ()
     {
+      var mockComparisonValue = new Mock<IComparable>(MockBehavior.Strict);
+      var mockPropertyValue = new Mock<IComparable>(MockBehavior.Strict);
+      mockPropertyValue.Setup(mock => mock.CompareTo(mockComparisonValue.Object)).Returns(3).Verifiable();
```

Comment:
Setups in usually use `_` as parameter name

```suggestion
      mockPropertyValue.Setup(_ => _.CompareTo(mockComparisonValue.Object)).Returns(3).Verifiable();
```